tests/unit/macros/test_adapters_macros.py

```diff
@@ -154,6 +176,7 @@ def test_macros_create_table_as_all_delta(self):
             "create or replace table my_table "
             "using delta "
             "partitioned by (partition_1,partition_2) "
+            "cluster by (cluster_1,cluster_2)"
```

Comment:
From the [docs](https://learn.microsoft.com/en-us/azure/databricks/delta/clustering#enable-liquid-clustering) (emphasis mine):

> Clustering is not compatible with **partitioning** or ZORDER, and requires that the Azure Databricks client manages all layout and optimization operations for data in your table.

Therefore the output of this `self._render_create_as()`, while expected, would technically raise an exception. Should we enforce any kind of block or just allow the compute cluster to raise an exception if the dbt user configures both `partition_by` and `liquid_clustered_by`?

Reply:
That's a good question. I'm for leaving the compute cluster to raise an exception, but would be open for suggestion

Reply:
My stance has been to let Databricks surface the issues, rather than intercepting in the adapter.  I'm not sure if we're doing this consistently everywhere.  My reasoning is that, for any rule we hard code in, the Databricks implementation could change, and it would be better for that functionality to be available to customers without us having to spin a new release.